NonWeb.md

```diff
@@ -3,15 +3,16 @@
 While WebAssembly is designed to run [within a browser](Web.md), it is
 also desirable for it to be able to execute well in other environments,
 including everything from minimal shells for testing to full-blown
-application environments e.g. on servers in datacenters. It may even be
-desirable to execute WebAssembly embedded within larger programs.
+application environments e.g. on servers in datacenters, or mobile/desktop
```

Comment:
Could you add "IoT" too? It's a wonderful buzzword, which the high-level goals also mention.